src/main/java/spireCafe/Anniv7Mod.java

```diff
@@ -247,6 +253,28 @@ public void receivePostInitialize() {
         BaseMod.addEvent(CafeRoom.ID, CafeRoom.class, "CafeDungeon");
         BaseMod.addCustomScreen(new CafeMerchantScreen());
         ConsoleCommand.addCommand("cafe", Cafe.class);
+
+        BaseMod.addSaveField("Anniv7DepletedPotion", new CustomSavable<List<Integer>>() {
```

Comment:
Could you move this into the `addSaveFields` method? You can see it being called above.